docs/build/circuits.ipynb

```diff
@@ -14,7 +14,10 @@
    "execution_count": null,
    "metadata": {
     "cellView": "form",
-    "id": "tUshu7YfcAAW"
+    "id": "tUshu7YfcAAW",
+    "vscode": {
+     "languageId": "python"
+    }
```

Comment:
Please remove the extra "vscode" metadata.

Reply:
Done